src/logic/schedule-parser/metadata.parser.ts

```diff
@@ -20,7 +20,7 @@ export class MetaDataParser extends MetadataProvider {
     } else {
       this.offset = 1;
       this.logLoadFIleError(
-        "Brak nagłówka z informacją o datach. Przyjęto, że pierwsza kolumna to pierwszy dzień miesiąca"
+        "Nie znaleziono nagłówka z informacją o datach. Jako pierwszy dzień miesiąca wczytano pierwszą kolumnę pliku."
```

Comment:
pliku -> grafiku

@kaplonpaulina 
pierwszą kolumnę - nie jest to prawda, bo pierwsza kolumna to nazwy pracowników

Reply:
to napisać `drugą kolumnę` (co paradoksalnie mimo prostoty niesie za sobą niebezpieczeństwo, że nie wszyscy zrozumieją prawidłowo XD), czy `pierwszą znalezioną kolumnę uzupełnioną wartościami zmian`? XD

Reply:
`pierwszą kolumnę uzupełnioną wartościami zmian`, nie brzmi najgorzej